addons/hr_timesheet/report/project_report_view.xml

```diff
@@ -28,7 +28,6 @@
                     <field name="allocated_hours" widget="timesheet_uom" type="measure"/>
                     <field name="effective_hours" widget="timesheet_uom" type="measure"/>
                     <field name="remaining_hours" widget="timesheet_uom" type="measure"/>
-                    <field name="total_hours_spent" widget="timesheet_uom" type="measure"/>
```

Comment:
we could just hide it, no? To avoid breaking xpath. 🙂 

(I know you change the xpath selected this field in the codebase but we could have custom code. 😬 )

```suggestion
                    <field name="total_hours_spent" widget="timesheet_uom" type="measure" invisible="1"/>
```

Reply:
I've tried, but the option would completely disappear from the 'Measures' dropdown menu if I did.

Reply:
ah true

Reply:
After discussing with @aju-odoo, we decided to not change that in stable, we will completely remove it in master. 🙂